ansible/roles/hand_e_icons/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+- name: Copy icons to /usr/local/share/icons
+  copy: src="files/icons" dest="/usr/local/share"
+  sudo: yes
+  sudo_user: "{{ros_user}}"
```

Comment:
do not use sudo use become

Reply:
Fixed in 46224876a8190b755bef8e977ff2492c4d3a05b4.